tests/test_cache.py

```diff
@@ -140,6 +140,8 @@ def test_abstract_methods(self):
             cache.clear_all()
         with self.assertRaises(NotImplementedError):
             cache.save(make_package())
+        with self.assertRaises(NotImplementedError):
+            cache.search({'name': [], 'summary': []}, 'or')
```

Comment:
It turns out that the exception is being raised by `self.distinct()` on line 201 in cache/base.py: https://github.com/stevearc/pypicloud/pull/107/files/a70236fc78730ca1e1091a9a7b18f395bdda8ac0#diff-4928d680f66969deaa7e1b09ab1c558cR201

This probably is not a good test, so I will take it out.